lib/news_scraper/scraper.rb

```diff
@@ -0,0 +1,23 @@
+module NewsScraper
+  class Scraper
+    def initialize(query:)
+      @query = query
+    end
+
+    def scrape
+      article_urls = Extractors::GoogleNewsRss.new(query: @query).extract
+
+      transformed_articles = []
```

Comment:
When we have a public doc, we'll have to ask the user to manually `rescue Transformers::ScrapePatternNotDefined` and deal with it. I'll add it to the README